classes/item.rb

```diff
@@ -0,0 +1,56 @@
+##
+# Item class takes 6 paramters
+# @genre is the categorie of item. ie: Music
+# @author is the owner of item. ie: book
+# @source
+# @label is the title of item. ie: album
+# @archived is the boolean value of item.
+# @publish_date is the published date of item.
+require 'date'
+class Item
+  attr_reader :genre, :author, :source, :label, :archived, :publish_date
+
+  def initialize(publish_date, archived: false)
+    @id = Random.rand(1..10_000)
+    @publish_date = Date.parse(publish_date)
+    @archived = archived
+  end
+
+  # add_genre adds a genre to the item
+  # @param genre is the genre object
+  def add_genre(genre)
+    @genre = genre
+  end
```

Comment:
- [ ] Let's really make sure that genre belongs to Item by adding this additional line of code(`genre.items.push(self) unless genre.items.include?(self)`). That is,


 ```.ruby
  def add_genre(genre)
    @genre = genre
    genre.items.push(self) unless genre.items.include?(self)
  end
```